llama_index/download/download_utils.py

```diff
@@ -27,10 +28,13 @@
 PATH_TYPE = Union[str, Path]
 
 
-def _get_file_content(loader_hub_url: str, path: str) -> Tuple[str, int]:
+def _get_file_content(loader_hub_url: str, path: str) -> Tuple[Union[str, bytes], int]:
     """Get the content of a file from the GitHub REST API."""
     resp = requests.get(loader_hub_url + path)
-    return resp.text, resp.status_code
+    if ".pdf" in path:
```

Comment:
better to add enum for such files in future.
.pdf could be one of them

Reply:
agreed! this needs cleaning up for sure